template/11-monetization.md

```diff
@@ -1,14 +1,18 @@
 # Business Model
 
-*Expected operating Costs*
+## Revenue Streams
 
-- Development costs  
-- Infrastructure costs: hosting, storage, and bandwidth  
-- Maintenance costs  
-- Marketing costs  
-- Operational costs: customer Support, administration, legal and compliance  
+Our business model for Proxima is mainly based on advertising for businesses of any sizes. Our main value proposition is providing localized-based content, allowing businesses of any size to reach their targeted audience. This enables a campaign of any size based on the live location of the user, offering from worldwide coverage to precise targeting down to districts or even neighborhoods for centralized actions. This can be viewed as a digital version of local billboards.
```

Comment:
```suggestion
Our business model for Proxima is mainly based on advertising for businesses of any sizes. Our main value proposition is providing localized-based content, allowing businesses of any size to reach their targeted audience. This enables campaigns of any size based on the live location of the user, ranging from worldwide coverage to precise targeting down to districts or even neighborhoods. This can be viewed as a digital version of local billboards.
```

Reply:
Thank you ! Added in: [bd7d338](https://github.com/ProximaEPFL/prd/pull/6/commits/bd7d338d80e0c4c4e4e2bc99aed20ca5e50cb658)